.github/workflows/evm-test.yml

```
@@ -0,0 +1,70 @@
+name: Forge Tests
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+        - "bridge-evm/**"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)